Content.Server/Worldgen/Tools/PoissonDiskSampler.cs

```diff
@@ -0,0 +1,194 @@
+﻿using Robust.Shared.Random;
+
+namespace Content.Server.Worldgen.Tools;
+
+/// <summary>
```

Comment:
I keep getting arithmetic overflows when I try to use this though not sure where my inputs were going wrong.

Reply:
it's a bit annoying but tl;dr the input bounds must be in a local coordinate space relative to 0,0.